generic/tests/yum_update.py

```diff
@@ -43,8 +43,9 @@ def run(test, params, env):
     timeout = int(params.get("login_timeout", 360))
     session = vm.wait_for_login(timeout=timeout)
 
-    internal_yum_update(session, "yum update", params.get("shell_prompt"), 600)
-    internal_yum_update(session, "yum update kernel",
+    internal_yum_update(session, "yum update --nogpgcheck",
+                        params.get("shell_prompt"), 600)
+    internal_yum_update(session, "yum update kernel --nogpgcheck",
```

Comment:
hi , i boot a 7.3 guest and check /etc/yum.repos.d/    the repo file is empty ,so that it will fail , when yum update -y
even if you add --nogpgcheck, since the repo is empty, it will still fail